README.md

```diff
@@ -68,6 +68,25 @@ This directory can include sub-directories to manage assets of similar type, and
 
 ## Usage
 
+### Home Layout
+
+`home.html` is a flexible HTML layout for the site's landing-page / home-page / index-page. <br/>
+
+#### Main Heading and Content-injection
+
+From Minima v2.2 onwards, the *home* layout will inject all content from your `index.md` / `index.html` **before** the **`Posts`** heading. This will allow you to include non-posts related content to be published on the landing page under a dedicated heading. *We recommended that you title this section with a Heading2 (`##`)*.
+
+Usually the `site.title` itself would suffice as the implicit 'main-title' for a landing-page. But, if your landing-page would like a heading to be explicitly displayed, then simply define a `title` variable in the document's front matter and it will be rendered with an `<h1>` tag.
```

Comment:
> When would you want the index's title to not be the site's title?

Very rarely. That's why it's optional. The `<h1>` tag won't render unless the user wants it to..
Would provide a fallback in the rare scenario that the user's custom stylesheet has a `.home h1` especially since `Posts` is now a `<h2>`

> It would also cause a conflict on GitHub Pages, which sets titles by default from the first heading.

I did consider this. Wouldn't the plugin not mangle the Index title if `page.title` is defined? I fail to see how a conflict would arise in this case.
The first heading previously  (`<h1>Posts</h1>`) is now after the `{{ content }}` which may or may not contain a heading. So this can be a solution for both situations.